### ISSUE TITLE
Your bowser isn't fully supported

### ISSUE THREAD

**Issue by jscurtu** [cross-platform/icloud-for-linux#3]
I get the following error when using iCloud Pages and co. 

![image](https://user-images.githubusercontent.com/590307/71089009-2eb02080-21a0-11ea-88ec-9768f72fcd48.png)

**Comment by MarcusTomlinson** [comment]
Indeed, Apple don't officially support a qml webview browser, but if you click Continue everything mostly works as it should anyway. The experience of having to click Continue is shoddy of course. I've got it on my todo to see if I can remove that.

**Comment by jscurtu** [comment]
Hi @MarcusTomlinson 

I think all you need to do is change the useragent with something supported by apple.. 
maybe something like this in the webview:

```
 profile:  WebEngineProfile{
        httpUserAgent: "Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:71.0) Gecko/20100101 Firefox/71.0"
    }
```

https://doc.qt.io/qt-5/qml-qtwebengine-webengineprofile.html#httpUserAgent-prop

https://developers.whatismybrowser.com/useragents/explore/

**Comment by MarcusTomlinson** [comment]
Ah! You're absolutely right. This will be fixed in 0.4 - releasing soon :) Thanks @jscurtu!
